doc/content/getting-started/migrating/major-changes.md

```diff
@@ -4,166 +4,174 @@ weight: 1
 
-Furthermore, {{% tts %}} brings full support for all LoRaWAN versions, as well as Class B and Class C modes.
+Furthermore, {{% tts %}} brings full support for all LoRaWAN versions, as well as modes to support class B and class C capabilities. 
 
 ## Rx1Delay
 
-Devices imported from {{% ttnv2 %}} are configured with an `Rx1Delay` of 1 second, by default. In {{% tts %}} we recommend using an `Rx1Delay` of 5 seconds to accommodate for high latency backhauls and/or [peering with Packet Broker]({{< ref "/reference/packet-broker" >}}). See the [MAC settings guide]({{< ref "/devices/mac-settings" >}}) for more information and instructions about configuring `Rx1Delay`.
+Devices imported from {{% ttnv2 %}} are configured with an `Rx1Delay` of 1 second, by default. In {{% tts %}} we recommend using an `Rx1Delay` of 5 seconds to accommodate for high latency backhauls and/or [peering with Packet Broker]({{< ref "/reference/packet-broker" >}}). 
+
+See the [MAC settings guide]({{< ref "/devices/mac-settings" >}}) for more information and instructions about configuring `Rx1Delay`.
 
 ## MAC Commands
 
 {{% tts %}} expects that all end devices comply with the LoRaWAN specification by default, which means that the end devices should respond to Network Server MAC command requests accordingly. If a device fails to answer a MAC Command in a timely manner, there may be disruptions to the device uplink or downlink traffic. As mentioned in the LoRaWAN specification, the Network Server of {{% tts %}} will always prioritize MAC commands over application payloads on downlink.
-Note, that in case a device is not fully compliant with the specification - it can still work on {{% tts %}}, but it may require custom MAC settings configuration.
+
+In case a device is not fully compliant with the LORaWAN specification, it can still work on {{% tts %}}, but it may require custom MAC settings configuration.
 
 ### DevStatusReq
 
 The `DevStatusReq` MAC command is sent by the Network Server periodically to check the current status of the end device. Devices are expected to send a `DevStatusAns` reply.
 
-For end devices that ignore this MAC command, make sure to configure the `StatusTimePeriodicity` (time duration after which a `DevStatusReq` is issued by the Network Server) and `StatusCountPeriodicity` (number of frames after which a `DevStatusReq` is issued) of the device explicitly to `0`. Same as with `Rx1Delay`, see the [MAC settings guide]({{< ref "/devices/mac-settings" >}}) for more information.
+For end devices that ignore this MAC command, make sure to configure the `StatusTimePeriodicity` (time duration after which a `DevStatusReq` is issued by the Network Server) and `StatusCountPeriodicity` (number of frames after which a `DevStatusReq` is issued) of the device explicitly to `0`. 
```

Comment:
What are these terms `StatusTimePeriodicity` and `StatusCountPeriodicity`? They are not TTS terms are they? Are they literally LoRaWAN?

Reply:
I have no idea, this is the existing content that someone else wrote. I was just tweaking it a bit